lib/miq_decorator.rb

```diff
@@ -11,6 +11,18 @@ def for(klass)
       decorator
     end
   end
+
+  delegate :fonticon, :to => :class
+  delegate :listicon_image, :to => :class
+
+  # As we are using Object#try on decorators, we need this override to avoid exceptions
+  def try(func, *args)
```

Comment:
(Which causes all services that have a picture to show `100/service.png` the tree :).)